crawl-ref/source/dat/descript/monsters.txt

```diff
@@ -1974,6 +2000,11 @@ A long and sinuous dragon, seemingly more neck and tail than anything else. Its
 skin shines like molten mercury, magical energies arc from its pointed snout,
 and it moves with terrifying swiftness.
 %%%%
+quicksilver elemental
+
+A mercurial spirit drawn from the elemental plane of earth, which exists in
+this world by inhabiting a shimmering body of liquid metal.
```

Comment:
Let's avoid run-ons:

```suggestion
A mercurial spirit drawn from the elemental plane of earth. It exists in
this world by inhabiting a shimmering body of liquid metal.
```

Also, huh, really pulling in the D&D/nethackisms for this one, huh? Not sure how I feel about an 'elemental plane of earth' suddenly entering the cosmology... suppose it doesn't do any harm.

Reply:
Earth elementals are already from there, so are our other elementals!

Reply:
Wow, i'd totally forgotten that! Need to brush up on my lore, clearly :)